### PR TITLE
#167484581 integrated authors haven with coveralls coverage service

### PR DESCRIPTION
**Pull request by placideirandora**
#### What does this PR do?
It integrates **Authors Haven** with **Coveralls** coverage service.

#### Description of Task to be done?
1. Setting up required dependencies/packages.
2. Adding coverage script.
3. Adding coverage badge in the readme.
3. Creating coveralls YAML file for coveralls integration configuration.

#### Screenshots
Screen Shot 2019-07-31 at 14 57 35

#### How should this be manually tested?
1. Clone this branch.
2. Do **npm install**.
3. View and check if you see the coverage badge with unknown status and also check in the readme if it links to the repo: coding-geeks-ah-backend

### CODE REVIEW

**Review by harerakalex — COMMENTED** [review]
# Looks good.

**Review by Jkadhuwa — APPROVED** [review]
Everything looks okay to me

**Comment by mkiterian** [comment]
@placideirandora, how do I get a coverage report locally?

**Comment by placideirandora** [comment]
> @placideirandora, how do I get a coverage report locally?

**Comment by placideirandora** [comment]
> @placideirandora, how do I get a coverage report locally?

There is a coverage script in package.json that you can run to get the coverage locally. You can invoke it by: **npm run coverage**

File: package.json [hunk 1]
```diff
@@ -4,9 +4,10 @@
   "description": "A Social platform for the creative at heart",
   "main": "index.js",
   "scripts": {
+    "dev": "nodemon --exec babel-node ./src/index.js",
+    "coverage": "nyc report --reporter=text-lcov | coveralls",
```
Comment (mkiterian) [inline package.json]: the coverage script does not run the tests, from which source is the report generated from?

**Review by mkiterian — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (placideirandora) [reply package.json]: The issue was a source of the tests. Now, It is working.

**Pull request merged**